Clean up docs

bahiirwa opened the pull request:
Add proper markdown hierarchy (in changelog also)

bahiirwa committed:
Update README.MD

bahiirwa committed:
Clean up Docs

bahiirwa committed:
Fixing local to live migration

Clean up docs

bahiirwa committed:
Images screenshots using wallet

bahiirwa committed:
Proper use of the tab navigation

bahiirwa committed:
Update with images

bahiirwa committed:
Clean up images linkages

justingreerbbi commented on README.MD:
```diff
@@ -1,39 +1,39 @@
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
+# User Wallet Credit System
+
+* Contributors: justingreerbbi
```
Please add your WordPress username to the contributor tag.

bahiirwa committed:
Fixing Issue #11

The code always displays the balance for each client. Just need to remove the options box in the admin options. Correct me if I am wrong. I did not see any options saved for this in DB

bahiirwa committed:
Update user-wallet.php

bahiirwa committed:
Update user-wallet.php
